API Blueprint Specification.md

```diff
@@ -766,30 +766,9 @@ Defined by the `Parameters` keyword written in a Markdown list item:
 #### Description
 Discussion of URI parameters in the _scope of the parent section_.
 
-This section **must** be composed of nested list items only. This section **must not** contain any other elements. One list item per URI parameter. The nested list items subsections inherit from the [Named section](#def-named-section) and are subject to additional formatting as follows:
+This section **must** be composed of nested list items only. This section **must not** contain any other elements. Each of the list item describes a single URI parameter using the **[Markdown Syntax for Object Notation][MSON] (MSON)**.
```

Comment:
Also I am not convinced that we should refer to MSON here. Align the syntax yes. But I would not claim it is MSON syntax as there are some gaps such as the required vs. optional or the fact that there are no nested types.

Also explaining URI syntax here without references to MSON would set a great building stone for introducing MSON later...